store/postgres/src/writable.rs

```diff
@@ -383,16 +383,16 @@ impl WritableStoreTrait for WritableAgent {
         Ok(())
     }
 
-    fn start_subgraph_deployment(&self, logger: &Logger) -> Result<(), StoreError> {
-        self.store.start_subgraph_deployment(logger)?;
+    async fn start_subgraph_deployment(&self, logger: &Logger) -> Result<(), StoreError> {
+        let store = self.store.cheap_clone();
+        let logger = logger.cheap_clone();
+        graph::spawn_blocking_allow_panic(move || store.start_subgraph_deployment(&logger))
+            .await
+            .map_err(Error::from)??;
```

Comment:
I had never seen `??` as a way to flatten a result, fun.

Reply:
Seems like it does the `Result<>` unwrap with return syntax desugaring twice :)